cmake/modules/FindBerkeleyDB.cmake

```diff
@@ -1,37 +1,110 @@
 # Try to find the BerkeleyDB librairies
-# BDB_FOUND - system has Berkeley DB lib
-# BDB_INCLUDE_DIR - the Berkeley DB include directory
-# BDB_LIBRARY - Library needed to use Berkeley DB
-# BDBXX_INCLUDE_DIR - the Berkeley DB include directory for C++
-# BDBXX_LIBRARY - Library needed to use Berkeley DB C++ API
+# BerkeleyDB_FOUND - system has Berkeley DB lib
+# BerkeleyDB_INCLUDE_DIR - the Berkeley DB include directory
+# BerkeleyDB_LIBRARY - Library needed to use Berkeley DB
 
 include(BrewHelper)
```

Comment:
Shouldn't this include also be executed only when CMAKE_HOST_APPLE is specified?